packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -265,6 +271,168 @@ contract DeployImplementationsOutput is BaseDeployIO {
         DeployUtils.assertValidContractAddress(address(_disputeGameFactoryImpl));
         return _disputeGameFactoryImpl;
     }
+
+    // -------- Deployment Assertions --------
+    function assertValidDeploy(DeployImplementationsInput _dii) public {
+        assertValidDelayedWETHImpl(_dii);
+        assertValidDisputeGameFactoryImpl(_dii);
+        assertValidL1CrossDomainMessengerImpl(_dii);
+        assertValidL1ERC721BridgeImpl(_dii);
+        assertValidL1StandardBridgeImpl(_dii);
+        assertValidMipsSingleton(_dii);
+        assertValidOpsmProxy(_dii);
```

Comment:
Q: Why is it that `opsm` is the only proxy in this list?

Reply:
I suppose it comes back to the point about the separation between DeploySuperchain and DeployImplementations being arbitrary.

Reply:
Yea, it's exactly this, it does feel a bit awkward / inconsistent, but it's ok for now